components/docs-chef-io/content/automate/ha_upgrade_introduction.md

```diff
@@ -79,4 +79,105 @@ We can also pass a flag in upgade command to avoid prompt for workspace upgrade.
+
+1. Do the backup configuration only when you have not provided the (backup information) configuration at the time of deployment. Refer backup section for [object storage](/automate/ha_backup_restore_aws_s3/).
+
+    {{< note >}}
+     During the deployment for the Primary and Secondary clusters, use the same S3 bucket name.
+    {{< /note >}}
+
+1. On Primary Cluster
+
+    - Take a backup of Primary cluster from bastion by running below command:
+
+    ```sh
+    chef-automate backup create --no-progress > /var/log/automate-backups.log
+    ```
+
+    - Create a bootstrap bundle; this bundle captures any local credentials or secrets that aren't persisted to the database. To create the bootstrap bundle, run the following command in one of the Automate nodes:
+
+    ```sh
+    chef-automate bootstrap bundle create bootstrap.abb
+    ```
+
+    - Copy `bootstrap.abb` to all Automate and Chef Infra frontend nodes in the Secondary cluster.
+
+
+1. On New AMI upgraded Cluster
+
+    - Install `bootstrap.abb` on all the Frontend nodes (Chef-server and Automate nodes) by running the following command:
+
+    ```cmd
+    sudo chef-automate bootstrap bundle unpack bootstrap.abb
+    ```
+
+    - Run the following command in bastion to get the ID of the backups:
+
+    ```sh
+    chef-automate backup list
+    ```
+
+    -On Secondary Cluster Trigger restore command `chef-automate backup restore` on one of the Chef Automate nodes.
+
+        - To run the restore command, you need the airgap bundle. Get the Automate HA airgap bundle from the location `/var/tmp/` in Automate instance. For example: `frontend-4.x.y.aib`.
+
+        - In case the airgap bundle is not present at `/var/tmp`, it can be copied from the bastion node to the Automate frontend node
+
+        - Run the command at the Automate node of Automate HA cluster to get the applied config:
+
+        ```bash
+        sudo chef-automate config show > current_config.toml
+        ```
+
+        - Add the OpenSearch credentials to the applied config. If using Chef Managed OpenSearch, add the config below into `current_config.toml` (without any changes).
+
+        ```bash
+        [global.v1.external.opensearch.auth.basic_auth]
+            username = "admin"
+            password = "admin"
+        ```
+
+        - In the New AMI Upgraded cluster, use the following sample command to restore the latest backup from any Chef Automate frontend instance.
+
+        ```cmd
+        id=$(chef-automate backup list | grep completed | tail -1 | awk '{print $1}')
+        sudo chef-automate backup restore <backup-url-to-object-storage>/automate/$id/ --patch-config /path/to/current_config.toml --airgap-bundle /var/tmp/frontend-4.x.y.aib --skip-preflight --s3-access-key "Access_Key"  --s3-secret-key "Secret_Key"
+        ```
+
+### Switch to New Upgraded Cluster
+
+Steps to switch to the New cluster are as follows:
```

Comment:
Include steps to re-configure user's DNS to new set of Loadbalances created in the `New cluster with upgraded AMIs`

Reply:
Done